pkgs/top-level/python-packages.nix

```diff
@@ -5308,7 +5308,7 @@ in {
   protobuf = callPackage ../development/python-modules/protobuf {
     disabled = isPyPy;
     doCheck = !isPy3k;
-    protobuf = pkgs.protobuf3_8;
+    protobuf = pkgs.protobuf3_12;
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/91446 for the protobuf version discussion.